scripts/fetch_places_and_screens.exs

```diff
@@ -405,11 +413,17 @@ stop_ids =
     %{"source_config" => %{"sources" => sources}} ->
       sources
 
-    %{"source_config" => [%{"sources" => top_sources}, %{"sources" => bottom_sources}]} ->
-      top_sources ++ bottom_sources
+    %{"source_config" => sources} ->
```

Comment:
@PaulJKim I had to make some changes to the paess logic to account for a new config shape. Can you look this over and let me know if I'm missing any or if you see some that aren't needed anymore?

Reply:
Sorry I missed this earlier! There's another field I think we'll need to account for which is `top_configs` which would be present if there's a `bottom_configs` field. You can copy over what I had actually ended up putting up a PR for a similar fix in #239 earlier this week 😅

Reply:
If you want, you can also bundle in the rest of those changes in that PR as well. One of them removes old logic that was resulting in duplicate PA/ESS signs being added to Screenplay, and the other one adds some missing/new custom labeling we need to apply to some sign zones.